src/locale/languages/ru.js

```diff
@@ -4,18 +4,22 @@ export const ru = {
       welcomeTitle: 'Добро пожаловать в GitPoint',
       welcomeMessage: 'Самый многофункциональный бесплатный GitHub клиент',
       notificationsTitle: 'Управление уведомлениями',
-      notificationsMessage: 'Просматривайте и управляйте всеми вашими непрочитанными и активными уведомлениями',
```

Comment:
Was there any changes in the `ru.js` file besides the formatting?
If not, it shouldn't have been part of this PR

Reply:
There simply made changes to the formatting, @nersoh you need to remove this change from PR.